example/README.md

```diff
@@ -45,7 +45,6 @@ import "package:objectbox/objectbox.dart";
 
 @Entity()
 class Note {
-    @Id()       // required; stored as a 64-bit unsigned integer in ObjectBox
```

Comment:
Should the README point out that using `@Id` is possible (e.g. doesn't this give the impression field must be named `id`)? Also that there is a naming convention if not using it?

Reply:
Never mind, #170 fixes this.